libs/blocks/global-footer/global-footer.js

```diff
@@ -302,7 +302,7 @@ class Footer {
             aria-label="${platform}"
             daa-ll="${getAnalyticsValue(platform, index + 1)}"
             target="_blank">
-            <svg xmlns="http://www.w3.org/2000/svg" class="feds-social-icon" alt="${platform} logo">
+            <svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" class="feds-social-icon" alt="${platform} logo">
```

Comment:
This looks wrong - we mark it as decorative via `aria-hidden`, but then we also add an unsupported `alt` attribute. The wrapping link has an `aria-label` attribute, we can remove the SVG's `alt` one.

Reply:
Fixed 👍